azure/iac/KeyVault.rego

```diff
@@ -217,4 +217,356 @@ enablePurgeProtection_metadata := {
     "Resource Type": "microsoft.keyvault/vaults",
     "Policy Help URL": "",
     "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/vaults"
+}
+
+
+# PR-AZR-ARM-KV-006
+
+default keyvault_Acl = null
+
+azure_attribute_absence ["keyvault_Acl"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.keyvault/vaults"
+    not resource.properties.networkAcls.defaultAction
```

Comment:
@shahinshirvani use **publicNetworkAccess** property to check enabled/disabled status.
Ref: https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/vaults?tabs=json

Reply:
@rezoan yes, I checked publicNetworkAccess  property in this rule PR-AZR-ARM-KV-010
in this rule PR-AZR-ARM-KV-006 I'm using defaultAction property deny/allow status
Ref: https://github.com/Azure/azure-policy/blob/master/built-in-policies/policyDefinitions/Key%20Vault/AzureKeyVaultFirewallEnabled_Audit.json

Reply:
@shahinshirvani cant we combine both PR-AZR-ARM-KV-010 and PR-AZR-ARM-KV-006 then into one rule in a or condition?
both are doing same thing.